src/lib/StringUtilities.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Converts strings of the form 'one-two-three' to 'One-Two-Three', also called 'train-case'.
+ */
+export function trainCase(value: string): string {
+    if (value) {
+        let newValue = '';
+
+        for(let i = 0; i < value.length; i++) {
+            newValue += (i === 0 || value[i - 1] === '-') 
+                ? value[i].toUpperCase()
+                : value[i];
+        }
+
+        return newValue;
+    }
+
+    return value;
+}
```

Comment:
No, "kebab-case" specifically converts to lower-case.  "Train-case" is similar but uses capitalized words.  `lodash` has a number of these types of conversion functions but not that specific one.